apps/src/sites/studio/pages/levels-aichat-main.js

```diff
@@ -0,0 +1,30 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import {getStore} from '@cdo/apps/redux';
+import {getStandaloneProjectId} from '@cdo/apps/lab2/projects/utils';
+import Lab2Wrapper from '@cdo/apps/lab2/views/Lab2Wrapper';
+import ProjectContainer from '@cdo/apps/lab2/projects/ProjectContainer';
+import ProgressContainer from '@cdo/apps/lab2/progress/ProgressContainer';
+import AichatView from '@cdo/apps/aichat/AichatView';
+import {logError} from '@cdo/apps/music/utils/MusicMetrics';
```

Comment:
@sanchitmalhotra126 Can you confirm that the desired approach is to create an `AichatMetrics` version of https://github.com/code-dot-org/code-dot-org/blob/0cda37a99938e07890ef0b3fb8f5b050d56f0170/apps/src/music/utils/MusicMetrics.ts#L4 here?

Reply:
Actually I've removed MusicMetrics in this PR: https://github.com/code-dot-org/code-dot-org/pull/52714, in favor of a global `Lab2MetricsReporter` singleton that any lab can use directly, so we shouldn't need a specific `AichatMetrics`.